src/tools/overlaybd-apply.cpp

```diff
@@ -54,10 +54,11 @@ int main(int argc, char **argv) {
     string commit_msg;
     string parent_uuid;
     std::string image_config_path, input_path, gz_index_path, config_path;
-    bool raw = false, verbose = false;
+    bool raw = false, mkfs = false, verbose = false;
 
     CLI::App app{"this is overlaybd-apply, apply OCIv1 tar layer to overlaybd format"};
     app.add_flag("--raw", raw, "apply to raw image")->default_val(false);
+    app.add_flag("--mkfs", mkfs, "mkfs before apply")->default_val(false);
```

Comment:
suggest implement in overlaybd-create.

Reply:
add --mkfs for both overlaybd-create and overlaybd-apply